dali/test/operators_graph.h

```diff
@@ -0,0 +1,25 @@
+#ifndef DALI_OPERATORS_GRAPH_H
+#define DALI_OPERATORS_GRAPH_H
+
+#include <string>
+
+namespace dali {
+
+namespace testing {
+
+struct Operator{};
+
+class OperatorsGraph {
+
+public:
+  OperatorsGraph(const char* operator_name) {}
+
+private:
+  std::vector<std::pair<Operator, std::set<size_t>>> graph_;
```

Comment:
What exactly is that supposed to do?

Reply:
It's supposed to be a graph of operators. Would you suggest better method to store the graph?

Reply:
How does this represent nodes with multiple inputs and multiple outputs?

Reply:
It's simple adjacency list. Vector of vertices, where every vertex contains an Operator (as an ID, it could be anything, int, string etc...) and indices of elements to which such node is adjacent (`std::set`, because for given 2 vertices there will be max 1 edge between them).

In other words you could say, that the graph would be `std::vector<std::set<size_t>>`, however we need mapping from vector idx to specific Operator.

Reply:
I would get rid of that file as we trying to express DALI graph in just different way.